src/java.base/share/man/java.1

```diff
@@ -5533,6 +5533,37 @@ application start\-up script with the same technique:
 .PP
 \f[CB]env\ JAVA_TOOL_OPTIONS=\-XX:+RecordDynamicDumpInfo\ bash\ app_start.sh\f[R]
 .RE
+.SS Using Dynamic CDS Archive File with \-XX:+AutoCreateSharedArchive
+.PP
+\f[CB]\-XX:+AutoCreateSharedArchive\f[R] is a more convenient way of
+creating/using CDS archives.
+Unlike the methods of manual CDS archive creation described in the
+previous section, with \f[CB]\-XX:+AutoCreateSharedArchive\f[R], it\[aq]s
+no longer necessary to have a separate trial run.
+Instead, you can always run the application with the same command\-line
+and enjoy the benefits of CDS automatically.
+.RS
+.PP
+\f[CB]java\ \-XX:+AutoCreateSharedArchive\ \-XX:SharedArchiveFile=hello.jsa\ \-cp\ hello.jar\ Hello\f[R]
+.RE
+.IP \[bu] 2
+If the specified archive file exists and is created by the same version
+of the JDK, the JVM will load this file as a dynamic archive.
+.IP \[bu] 2
+Otherwise the JVM will not load the specified archive.
+Instead, it will automatically create (or update) the archive file at
+exit.
```

Comment:
The use of "otherwise" and "instead" is unclear in the context of an existing archive file. What exactly will happen?
1. If the file exists and was created by the same JDK version then it will be used.
2. Regardless of (1) the named archive file will be written at VM exit, ready for use the next time?
Is (2) correct even if the file exists and is from a different JDK version? Or do we only write the file if it either didn't exist, or is from the same JDK version?